src/page/ConfigPageLayout/tabs/TerraformTab.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect } from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { Alert, Text, TextLink, useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+import { generateRoutePath } from 'routes';
+
+import { ROUTES } from 'types';
+import { FaroEvent, reportEvent } from 'faro';
+import { useTerraformConfig } from 'hooks/useTerraformConfig';
+import { Clipboard } from 'components/Clipboard';
+
+import { ConfigContent } from '../ConfigContent';
+
+export function TerraformTab() {
+  const { config, checkCommands, probeCommands, error, isLoading } = useTerraformConfig();
+  const styles = useStyles2(getStyles);
+  useEffect(() => {
+    reportEvent(FaroEvent.SHOW_TERRAFORM_CONFIG);
+  }, []);
+
+  if (isLoading) {
+    return <ConfigContent loading={isLoading} title="Terraform config" />;
+  }
+
+  return (
+    <ConfigContent title="Terraform config">
+      {error && <Alert title={error.message} />}
+      <div>
+        <p>
+          You can manage Synthetic monitoring checks using Terraform as well as export your current checks as
+          configuration.
+        </p>
+      </div>
+      <div>
+        <h5>Prerequisites</h5>
```

Comment:
Just leave this comment for the headings to bump up for this file.

![Screenshot of the HeadingsMap for the Terraform page in Synthetic Monitoring. It highlights two headings with the wrong elements being used.](https://github.com/user-attachments/assets/2726f63e-fb42-4e86-b9a2-c7cd0bb428f7)